projects/examples/src/components/form-input/form-select-disabled.example.component.html

```diff
@@ -0,0 +1,9 @@
+<form [formGroup]="formGroup" class="clr-form-horizontal">
```

Comment:
Please add some text explaining the feature. For example


Options within the select element can be disabled by passing in disabled property. See  {@link SelectOption#disabled}